pymc/distributions/simulator.py

```diff
@@ -66,7 +66,7 @@ def sum_stat(cls, *args, **kwargs):
 class Simulator(NoDistribution):
     r"""
     Simulator distribution, used for Approximate Bayesian Inference (ABC)
-    with Sequential Monte Carlo (SMC) sampling via ``pm.sample_smc``.
+    with Sequential Monte Carlo (SMC) sampling via ``~pymc.sample_smc``.
```

Comment:
```suggestion
    with Sequential Monte Carlo (SMC) sampling via :func:`~pymc.sample_smc`.
```

it needs to use this syntax in order to be rendered as a link, not only use the correct import path of the function. Ref: https://pymc-data-umbrella.xyz/en/latest/about/contributing_to_documentation/docs_presentation.html#cross-referencing-roles